openstack-mixin/panels.libsonnet

```diff
@@ -0,0 +1,1004 @@
+          },
+          '1': {
+            color: 'green',
+            index: 0,
+            text: 'Up',
+          },
+        },
+      }),
+
+    novaStatus:
+      commonlib.panels.generic.stat.base.new(
+        'Nova status',
+        targets=[t.novaStatus],
+        description='Reports the status of the Nova compute service.',
+      )
+      + stat.standardOptions.withUnit('string')
+      + stat.options.withGraphMode('none')
+      + stat.standardOptions.withMappings({
+        type: 'value',
+        options: {
+          '0': {
+            color: 'red',
+            index: 1,
+            text: 'Down',
+          },
+          '1': {
+            color: 'green',
+            index: 0,
+            text: 'Up',
+          },
+        },
+      }),
+
+    neutronStatus:
+      commonlib.panels.generic.stat.base.new(
+        'Neutron status',
+        targets=[t.neutronStatus],
+        description='Reports the status of the Neutron networking service.',
+      )
+      + stat.standardOptions.withUnit('string')
+      + stat.options.withGraphMode('none')
+      + stat.standardOptions.withMappings({
+        type: 'value',
+        options: {
+          '0': {
+            color: 'red',
+            index: 1,
+            text: 'Down',
+          },
+          '1': {
+            color: 'green',
+            index: 0,
+            text: 'Up',
+          },
+        },
+      }),
+
+    cinderStatus:
+      commonlib.panels.generic.stat.base.new(
+        'Cinder status',
+        targets=[t.cinderStatus],
+        description='Reports the status of the Cinder block storage service.',
+      )
+      + stat.standardOptions.withUnit('string')
+      + stat.options.withGraphMode('none')
+      + stat.standardOptions.withMappings({
+        type: 'value',
+        options: {
+          '0': {
+            color: 'red',
+            index: 1,
+            text: 'Down',
+          },
+          '1': {
+            color: 'green',
+            index: 0,
+            text: 'Up',
+          },
+        },
+      }),
+
+    glanceStatus:
+      commonlib.panels.generic.stat.base.new(
+        'Glance status',
+        targets=[t.glanceStatus],
+        description='Reports the status of the Glance image service.',
+      )
+      + stat.standardOptions.withUnit('string')
+      + stat.options.withGraphMode('none')
+      + stat.standardOptions.withMappings({
+        type: 'value',
+        options: {
+          '0': {
+            color: 'red',
+            index: 1,
+            text: 'Down',
+          },
+          '1': {
+            color: 'green',
+            index: 0,
+            text: 'Up',
+          },
+        },
+      }),
+
+    alertsPanel:
+      alertList.new('Alerts')
+      + alertList.panelOptions.withDescription('Panel to report on the status of firing alerts.')
+      + alertList.options.UnifiedAlertListOptions.withAlertInstanceLabelFilter(this.grafana.variables.queriesGroupSelectorAdvanced),
+
+    totalResources:
```

Comment:
No data on this one.

Reply:
@Caleb-Hurshman read your comment. Got it to come in so I'm good here.